test/tests/jobs/UpdateJobTest.cpp

```diff
@@ -0,0 +1,53 @@
+#include <test/lib/BedrockTester.h>
+#include <test/tests/jobs/JobTestHelper.h>
+
+struct UpdateJobTest : tpunit::TestFixture {
+    UpdateJobTest()
+            : tpunit::TestFixture("UpdateJob",
+                                  BEFORE_CLASS(UpdateJobTest::setupClass),
+                                  TEST(UpdateJobTest::updateJob),
+                                  AFTER(UpdateJobTest::tearDown),
+                                  AFTER_CLASS(UpdateJobTest::tearDownClass)) { }
+
+    BedrockTester* tester;
+
+    void setupClass() { tester = new BedrockTester(_threadID, {{"-plugins", "Jobs,DB"}}, {});}
+
+    // Reset the jobs table
+    void tearDown() {
+        SData command("Query");
+        command["query"] = "DELETE FROM jobs WHERE jobID > 0;";
+        tester->executeWaitVerifyContent(command);
+    }
+
+    void tearDownClass() { delete tester; }
+
+    // Simple UpdateJob with all parameters
+    void updateJob() {
+        // Create the job
+        SData command("CreateJob");
+        string jobName = "job";
+        command["name"] = jobName;
```

Comment:
shouldn't you set the priority here too? to 500 or something? then after you update it do an assert that `oldPriority != new Priority`